### PR TITLE
Fix date in email.

### PR DESCRIPTION
**Pull request by MrZammler**
##### Summary

Fix the date in the email.

##### Component Name

Health

### CODE REVIEW

File: health/notifications/alarm-notify.sh.in [hunk 1]
```diff
@@ -2669,10 +2654,10 @@ if [ -n "$total_warn_alarms" ]; then
    while read -d, -r pair; do
        IFS='=' read -r key val <<<"$pair"
 
-       date=$(date --date=@${val} "${date_format}" 2>/dev/null)
-       [ -z "${date}" ] && date=$(date "${date_format}" 2>/dev/null)
-       [ -z "${date}" ] && date=$(date --date=@${val} 2>/dev/null)
-       [ -z "${date}" ] && date=$(date 2>/dev/null)
+       date_w=$(date --date=@${val} "${date_format}" 2>/dev/null)
```
Comment (ilyam8) [inline health/notifications/alarm-notify.sh.in:2657]: what does `_w` mean?

Comment (MrZammler) [reply health/notifications/alarm-notify.sh.in:2657]: Hi Ilya, It's the date of the warning... I was overriding the date variable before, it needs to be a different var.

Comment (ilyam8) [reply health/notifications/alarm-notify.sh.in:2657]: i see, was about to suggest to name it `warning_date` (or `date_warning`) 😄

Comment (MrZammler) [reply health/notifications/alarm-notify.sh.in:2657]: We could yes :-)

Many of these will go through proper review when they go into master, so they can/will be changed !

**Pull request merged**